### ISSUE TITLE
(Discuss complications) Create variable table-based step to allow testing a 'story'

### ISSUE THREAD

**Issue by plocket** [plocket/docassemble-cucumber#77]
```
And the data is
    | var | choice | value |
    | acknowledged_information_use | user_accepts | true |
    | person_answering |  | tenant |
```

How do we hit a 'continue' button only on appropriate pages?

**Comment by plocket** [comment]
The current solution is to continue whenever there aren't any variables left on the page.

**Comment by plocket** [comment]
Apparently I made a bunch of other issues instead of continuing to write in this one. This was closed by #92.
